XpressAI/xircuits
# ✨ Update Literal List, Dict, and Tuple to use TextArea Input

# Description

This pull request updates the interface for Literal List, Dict, and Tuple to use the `TextAreaInput` dialogue, previously only used by Literal Strings, to allow for easier editing of longer inputs.

## Pull Request Type


## Type of Change


# Tests

1. Create a new instance of `LiteralInputDialog` specifying different types (e.g., string, dict).
2. Verify that the header and example text adjust according to the specified type.
3. Confirm that input behavior work as expected.

## Tested on?
